packages/gafl-webapp-service/src/pages/contact/contact/contact.njk

```diff
@@ -3,23 +3,24 @@
 {% from "input/macro.njk" import govukInput %}
 {% from "warning-text/macro.njk" import govukWarningText %}
 
-{% set title = 'How would you like to receive your licence details?' if data.isPhysical else 'What is your preferred way of being contacted?' %}
-{% set errorMsg = 'You have not told us how would you like to receive your licence details?' if data.isPhysical else 'You have not told us your preferred way of being contacted?' %}
-{% set errorMsg2 = 'Select how would you like to receive your licence details?' if data.isPhysical else 'Select your preferred way of being contacted?' %}
+{% set title = 'How should we contact you?' if data.isPhysical else 'How should we send you the licence?' %}
+{% set errorMsg = 'Choose how you would like to be contacted' if data.isPhysical else 'Choose how you would like to receive the licence details' %}
+{% set emailError = 'Enter an email address in the correct format, like name@example.com' %}
+{% set mobileError = "Enter a mobile number, for overseas numbers include '+' followed by your country code" %}
```

Comment:
We don't send SMS messages to overseas numbers - there is a ticket to address the validator for this.

Reply:
This still needs to be addressed.